MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -148,6 +157,41 @@ private void WorkerThread()
                     //Process a single incoming byte from the client session
                     session.ProcessDataFromClient();
 
+                    //Global Command Handler
+                    if (session.Status == 3 && DoGlobalsAttribute.Get(session.SessionState))
```

Comment:
future todo, a descriptive enum instead of 3 would go a long way :)

Reply:
Agreed -- we should probably create an EnumChannelStatus to track the different codes and give some documentation on their meanings